requirements.txt

```diff
@@ -0,0 +1,69 @@
+annotated-types==0.5.0
```

Comment:
Please do not use== since we May have problema in different setup on different distributions

Reply:
Please also reduce the list to the minimum set, the subdependencies should be resolved by pip

Reply:
Removed requirements.txt as we agreed in call.
Commit f71e84f994f7b5ecb934c55683159972a3ff67be